File: src/main/java/com/tianli/auth/grasscutterauth/mapper/LoginMapper.java
package com.tianli.auth.grasscutterauth.mapper;

import com.tianli.auth.grasscutterauth.domain.User;
import org.apache.ibatis.annotations.Select;

public interface LoginMapper {

    @Select("SELECT * FROM auth_user WHERE id = #{id}")
    User getUserById(Long id);

}

File: src/main/java/com/tianli/auth/grasscutterauth/mapper/LoginMapper.kt
package com.tianli.auth.grasscutterauth.mapper;
import com.tianli.auth.grasscutterauth.domain.User
import org.apache.ibatis.annotations.Select

interface LoginMapper {

    @Select("SELECT * FROM auth_user WHERE id = #{id}")
    fun getUserById(id:Long?): User;

}
